index.js

```diff
@@ -30,6 +30,26 @@ Cypress.Commands.add('upload', { prevSubject: 'element' }, (subject, fileOrArray
       processedFiles.forEach(file => dataTransfer.items.add(file));
       const input = subject[0];
       input.files = dataTransfer.files;
+
+      if (isManualTriggerRequired()) {
+        return cy.wrap(subject).trigger('change');
+      }
+
+      return null;
+    }
+
+    function isManualTriggerRequired() {
```

Comment:
Here is the hackish solution we agreed on.
I'll make a release of this to ease testing process. Any ideas/suggestions/bug reporting is welcome to the related issue: #34